UnityProject/Assets/Scripts/Weapons/Projectiles/BulletBehaviour.cs

```diff
@@ -27,7 +28,17 @@ private void StartShoot(Vector2 dir, float angle, GameObject controlledByPlayer,
 		thisRigi.AddForce(dir.normalized * 24f, ForceMode2D.Impulse);
 	}
 
-	public abstract void OnShoot();
+
+    //TODO
+    /**
+     * This function should be converted to a standard non abstract function as to  
+     *      Move forward with making the guns be the deteriminers of damage. This means you will
+     *      go by Gun's damage value and type rather then the bullet. Note we probably want to store/use the weapon that
+     *      does the firing here some how. 
```

Comment:
The gun already determines the damage, As I told you.
The gun gives the damage value to the bullet. 

So this should be removed as it's wrong. Bullets don't have a a damage value of themselves, look at the gun prefab for damage values